packages/notifications/push/amplify_push_notifications/android/src/main/kotlin/com/amazonaws/amplify/amplify_push_notifications/PushNotificationsHostApiBindings.java

```diff
@@ -1,7 +1,7 @@
 // 
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-// Autogenerated from Pigeon (v7.2.1), do not edit directly.
+// Autogenerated from Pigeon (v9.2.4), do not edit directly.
```

Comment:
This one might be worth renaming or adjusting the `.gitattributes` for. Does not have to be in this PR though. Out of scope for the changes here.

At some point we should also look into using kotlin. Maybe we update it than as I think file naming in kotlin is less restrictive than in Java.

Reply:
Agreed and agreed.

Kotlin and Swift were both recently upgraded from "experimental" to stable in pigeon.